packages/docs/src/routes/components/fluidgrid.md

```diff
@@ -0,0 +1,67 @@
+---
+title: FluidGrid
+description: Svelte Materialify supports the 12 point Material Design grid for laying out and controlling breakpoints for your application.
+keywords: grids, svelte materialify grid, layout component, flex component
+related:
+  - components/container
+  - components/row
+  - components/col
+---
+
+<script>
+  import Playground from '@/playground/FluidGrid.svelte';
+</script>
+
+<style lang="scss">
+  @mixin make-item-rule {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
```

Comment:
I'm unsure of `height: 100%` (and maybe `align-items: center`) has to go into `FluidGrid.scss` for `stretch` to work properly.

Reply:
I don't see any difference without it, also it would be an easy change if someone has problems with it :)